### PR TITLE
Feat/prevent string in number field

### LINKED ISSUE #815: ui: fix minutes error message style on badge creation
After entering an invalid number, the error message style is broken
Image

### PR DESCRIPTION
**Pull request by Thiemann96**
- strings are not allowed anymore 
 - other error messages are displayed as <p> at the bottom of the input field

### CODE REVIEW

File: src/app/components/input.error.component.ts [hunk 1]
```diff
@@ -7,19 +7,17 @@ import { HlmInputErrorDirective } from './spartan/ui-input-helm/src';
 	standalone: true,
 	imports: [HlmInputErrorDirective],
 	host: {
-		class: 'tw-block tw-mt-0 md:tw-mt-1 tw-min-h-[20px] tw-mb-4',
+		class: 'tw-block tw-mt-0 md:tw-mt-1 tw-min-h-[20px] tw-mb-4 overflow-visible',
```
Comment (jona159) [inline src/app/components/input.error.component.ts:10]: ```tw-``` prefix missing

File: src/app/components/input.error.component.ts [hunk 2]
```diff
@@ -7,19 +7,17 @@ import { HlmInputErrorDirective } from './spartan/ui-input-helm/src';
...
 		// eslint-disable-next-line @angular-eslint/no-host-metadata-property
 		// '[class.invisible]': "touchedState() === 'UNTOUCHED'",
 	},
-	template: `
-		<p>{{ error }}</p>
-	`,
+	template: ` <p class="tw-whitespace-nowrap">{{ error }}</p> `,
```
Comment (jona159) [inline src/app/components/input.error.component.ts:14]: Have you tested if this breaks the layout for long error messages?

**Review by jona159 — COMMENTED** [review]
Please check the comments and the merge conflict

File: src/app/components/input.error.component.ts (hunk 1 above)
Comment (Thiemann96) [reply src/app/components/input.error.component.ts]: wasn't used/needed anyway - i removed it

**Comment by Thiemann96** [comment]
@jona159 I moved this PR to [a new one](https://github.com/mint-o-badges/badgr-ui/pull/887) for readability and merge conflicts I encountered

**Pull request closed without merging**